history-of-quakers/modernized/05-chapter-3.adoc

```diff
@@ -103,19 +115,20 @@ And now the priests began to prophesy again, that within half a year,
 the Quakers should all be put down and gone.
 But they reckoned wrong; for it fared with those people as with trees,
 which grow best when most lopped.
-Duris ut ilex tonsa pipennibus, perdamna, per caedes, ab ipso,
-ducit opes animumque ferro.^
+__Duris ut ilex tonsa pipennibus, perdamna, per caedes, ab ipso,
+ducit opes animumque ferro.__^
 footnote:[IMITATED.
 {footnote-paragraph-split}
+
+[verse]
```

Comment:
So here is a verse in the footnotes. I wasn't sure if I could use the verse formatting here, which i see I can't, but I thought I'd leave it so you could see it and tell me what to do.

Reply:
yeah, footnote poetry is probably the hardest syntax, and i didn't even put it in the styleguide.  you can leave this for now, but I'll come back and fix it after I merge.  it's rare enough that I think I'll just handle them when they come up.  if it turns out that there are a bunch in this book, maybe i'll add it to the styleguide -- although it's tough because you still can't see them. 🤔